src/main/java/com/github/deeepamin/ciaid/services/injectors/AbstractGitlabCIYamlScriptInjector.java

```diff
@@ -34,18 +48,18 @@ public void getLanguagesToInject(@NotNull MultiHostRegistrar registrar, @NotNull
         LOG.debug(String.format("Script %s is not sequence item, skipping injection", context.getText()));
         return;
       }
-      injectShell(registrar, context);
+      injectShell(registrar, context, language);
     }
     if (context instanceof YAMLScalarList || context instanceof YAMLScalarText) {
-      injectShell(registrar, context);
+      injectShell(registrar, context, language);
     }
   }
 
-  private void injectShell(MultiHostRegistrar registrar, PsiElement context) {
+  private void injectShell(MultiHostRegistrar registrar, PsiElement context, @NotNull Language language) {
```

Comment:
Passing `language` as a parameter to be able to make it `@NonNull`. The caller checked for nullness.